releases/v1.1-alpha.20170803.md

```diff
@@ -0,0 +1,87 @@
+---
+title: What&#39;s New in v1.1-alpha.20170803
+toc: false
+summary: Additions and changes in CockroachDB version v1.1-alpha.20170803
+redirect_from: /v1.1-alpha.20170803.html
+---
+
+## Aug 3, 2017
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170803.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170803.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170803.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170803.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Backwards-Incompatible Changes
+
+- Table and column names in double quotes are now case-sensitive (same as in PostgreSQL). [#16884](https://github.com/cockroachdb/cockroach/pull/16884)
```

Comment:
It may be useful to add: `Any existing use of tables in the system database will need to quote column names containing upper case letters.`